addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -740,6 +739,49 @@
         // Private
         //---------------------------------------------------------------------
 
+        _getSlide: function (slideId) {
```

Comment:
Would be nice to explain a bit more in docstring why we need this.
(e.g: DOM structure is different for slides of a "documentation" channel etc etc).